libunwind/include/unwind_itanium.h

```diff
@@ -24,7 +24,7 @@ struct _Unwind_Exception {
                             _Unwind_Exception *exc);
 #if defined(__SEH__) && !defined(__USING_SJLJ_EXCEPTIONS__)
   uintptr_t private_[6];
-#else
+#elif !defined(__USING_WASM_EXCEPTIONS__)
```

Comment:
Is this a macro that already exists somewhere in libunwind ?

Reply:
No we add it to the command line both for [libcxxabi](https://github.com/emscripten-core/emscripten/blob/e1236625707595d253c189c5b152ed809afc54a5/tools/system_libs.py#L1506) and [libunwind](https://github.com/emscripten-core/emscripten/blob/e1236625707595d253c189c5b152ed809afc54a5/tools/system_libs.py#L1606).